README.md

```diff
@@ -93,6 +93,19 @@ When elements are extracted from PDFs or images, it may be useful to get their b
   | jq -C . | less -R
 ```
 
+#### Encoding
+
+You can specify the encoding to use to decode the text input. If no value is provided, utf-8 will be used.
+
+```
+curl  -X 'POST'  'http://localhost:8000/general/v0/general'  \
```

Comment:
```suggestion
curl -X 'POST' \
  'https://api.unstructured.io/general/v0/general' \
```
usual we add our api to readme because it will soon be deployed to prod (after this pr is merged)